Orange/util.py

```diff
@@ -3,20 +3,25 @@
 from functools import wraps
 from itertools import chain, count
 from collections import OrderedDict
-import logging
+import warnings
 
 import numpy as np
 
 
-log = logging.getLogger()
+warnings.simplefilter('default', DeprecationWarning)
```

Comment:
I think this would set the filter for all DeprecationWarnings not just the ones raised by `deprecated` (or was this the intention). I think you can just subclass a DeprecationWarning to be raised by the decorator and set the default filter on that.